src/state/TileStore.ts

```diff
@@ -201,7 +202,7 @@ export class TileStoreBuilder {
       if (
         media === this.prevSpotlightSpeaker &&
         this.spotlight.media.length === 1 &&
-        "speaking" in this.spotlight.media[0] &&
```

Comment:
would a `this.spotlight.media[0]?.speaking$ !== undefined` have helped here? Would ts let us know that `speaking` is not available anyhow...